Budget Planner/graph.js

```diff
@@ -0,0 +1,178 @@
+const dims={height:300,width:300,radius:150};
+const cent ={x:(dims.width/2 + 5), y:(dims.height / 2 + 5)};
+
```

Comment:
Please make spacing consistent (here around = and { signs), throughout the files.